src/main/java/io/confluent/connect/jdbc/sink/JdbcSinkTask.java

```diff
@@ -74,14 +74,18 @@ public void put(Collection<SinkRecord> records) {
           remainingRetries,
           sqle
       );
+      String sqleAllMessages = "";
+      for (Throwable e : sqle) {
+        sqleAllMessages += e.getMessage() + "\n";
```

Comment:
We should probably use the `toString` method of the throwable, because if it doesn't have a cause "null" will be printed. So this would be: 

`sqleAllMessages += e + "\n";`

If `cause` is null, it'll at least print the classname of the exception.

Reply:
Also, I've assumed that we expect a small number of exceptions linked within the `SQLException`, that's why we concatenate the string in a loop. If that's not always the case we should use a `StringBuilder` instead.

Reply:
That makes sense. I'll change it.

Yes we deployed this fix to our prod system, there are usually 2, 3 exceptions linked in the SQLException. Not a lot of them.